metaseq_cli/train.py

```diff
@@ -93,18 +142,24 @@ def main(cfg: DictConfig) -> None:
     assert cfg.criterion, "Please specify criterion to train a model"
 
     # Build model and criterion
+    built_model = task.build_model(cfg.model)
+    if cfg.distributed_training.tp_enabled:
```

Comment:
for 175B model, we need to use meta tensor to initialize the model. Not for June